e2e/__tests__/dependencyClash.test.ts

```diff
@@ -7,17 +7,14 @@
 
 import * as path from 'path';
 import {tmpdir} from 'os';
-import {skipSuiteOnWindows} from '@jest/test-utils';
 import {cleanup, createEmptyPackage, writeFiles} from '../Utils';
 import runJest from '../runJest';
 
-skipSuiteOnWindows();
-
 // doing test in a temp directory because we don't want jest node_modules affect it
 const tempDir = path.resolve(tmpdir(), 'clashing-dependencies-test');
-const hasteImplModulePath = path.resolve(
```

Comment:
`path.posix.resolve` instead of the `replace`

Reply:
Very good point!

Reply:
Ok so turns out have to use `replace`, as [`path.posix.resolve` doesn't care that you want posix](https://github.com/nodejs/node/issues/13887) 😬